test/SerialIOTest.cpp

```diff
@@ -1087,10 +1089,407 @@ TEST_CASE( "no_serial_hdf5", "[serial][hdf5]" )
+
+    for( uint64_t i = 0; i < 4; ++i )
+    {
+        *position_local_1 = position_global[i];
+        e_1["position"]["x"].storeChunk({i}, {1}, position_local_1);
+        o.flush();
+    }
+
+    std::vector< uint64_t > positionOffset_global(4);
+    uint64_t posOff{0};
+    std::generate(positionOffset_global.begin(), positionOffset_global.end(), [&posOff]{ return posOff++; });
+    std::shared_ptr< uint64_t > positionOffset_local_1(new uint64_t);
+    e_1["positionOffset"]["x"].resetDataset(Dataset(determineDatatype(positionOffset_local_1), {4}));
+
+    for( uint64_t i = 0; i < 4; ++i )
+    {
+        *positionOffset_local_1 = positionOffset_global[i];
+        e_1["positionOffset"]["x"].storeChunk({i}, {1}, positionOffset_local_1);
+        o.flush();
+    }
+
+    ParticleSpecies& e_2 = o.iterations[2].particles["e"];
+
+    std::generate(position_global.begin(), position_global.end(), [&pos]{ return pos++; });
+    std::shared_ptr< double > position_local_2(new double);
+    e_2["position"]["x"].resetDataset(Dataset(determineDatatype(position_local_2), {4}));
+
+    for( uint64_t i = 0; i < 4; ++i )
+    {
+        *position_local_2 = position_global[i];
+        e_2["position"]["x"].storeChunk({i}, {1}, position_local_2);
+        o.flush();
+    }
+
+    std::generate(positionOffset_global.begin(), positionOffset_global.end(), [&posOff]{ return posOff++; });
+    std::shared_ptr< uint64_t > positionOffset_local_2(new uint64_t);
+    e_2["positionOffset"]["x"].resetDataset(Dataset(determineDatatype(positionOffset_local_2), {4}));
+
+    for( uint64_t i = 0; i < 4; ++i )
+    {
+        *positionOffset_local_2 = positionOffset_global[i];
+        e_2["positionOffset"]["x"].storeChunk({i}, {1}, positionOffset_local_2);
+        o.flush();
+    }
+
+    o.flush();
+
+    ParticleSpecies& e_3 = o.iterations[3].particles["e"];
+
+    std::generate(position_global.begin(), position_global.end(), [&pos]{ return pos++; });
+    std::shared_ptr< double > position_local_3(new double);
+    e_3["position"]["x"].resetDataset(Dataset(determineDatatype(position_local_3), {4}));
+
+    for( uint64_t i = 0; i < 4; ++i )
+    {
+        *position_local_3 = position_global[i];
+        e_3["position"]["x"].storeChunk({i}, {1}, position_local_3);
+        o.flush();
+    }
+
+    std::generate(positionOffset_global.begin(), positionOffset_global.end(), [&posOff]{ return posOff++; });
+    std::shared_ptr< uint64_t > positionOffset_local_3(new uint64_t);
+    e_3["positionOffset"]["x"].resetDataset(Dataset(determineDatatype(positionOffset_local_3), {4}));
+
+    for( uint64_t i = 0; i < 4; ++i )
+    {
+        *positionOffset_local_3 = positionOffset_global[i];
+        e_3["positionOffset"]["x"].storeChunk({i}, {1}, positionOffset_local_3);
+        o.flush();
+    }
+
+    o.flush();
+}
+
+TEST_CASE( "hzdr_adios1_sample_content_test", "[serial][adios1]" )
+{
+    // since this file might not be publicly available, gracefully handle errors
```

Comment:
please add `/** @todo add bp example files to https://github.com/openPMD/openPMD-example-datasets */`